prov/efa/src/efa_hmem.c

```diff
@@ -11,7 +11,7 @@ struct efa_hmem_info g_efa_hmem_info[OFI_HMEM_MAX];
 static size_t efa_max_eager_msg_size_with_largest_header() {
 	int mtu_size;
 
-	mtu_size = g_device_list[0].rdm_info->ep_attr->max_msg_size;
```

Comment:
Need to make sure the old rdm_info->ep_attr->max_msg_size doesn't cut the header size, otherwise the protocol threshold calculation could be wrong by this change.

Reply:
It does not https://github.com/ofiwg/libfabric/blob/c3420a7402a10f5fe3f475ac5eb39b3f2837f8d1/prov/efa/src/efa_prov_info.c#L194